From bcf3007d1ed2e501764a5e2667337651154dc75b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 25 Sep 2020 02:01:33 -0700
Subject: [PATCH 1667/1851] ASoC: xlnx: formatter: Fix warnings for
 UNUSED_VALUE

commit ae919a4a2f8d5db7605159f147bd30e58f59b4f4 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the UNUSED_VALUE type.

The actual warning for below function is:

For xlnx_formatter_pcm_hw_params():

Event value_overwrite: 	Overwriting previous write to "bits_per_sample"
with value from "params_width(params)".

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xlnx_formatter_pcm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/xilinx/xlnx_formatter_pcm.c b/sound/soc/xilinx/xlnx_formatter_pcm.c
index d02120310d2c..277637c13a80 100644
--- a/sound/soc/xilinx/xlnx_formatter_pcm.c
+++ b/sound/soc/xilinx/xlnx_formatter_pcm.c
@@ -464,7 +464,6 @@ static int xlnx_formatter_pcm_hw_params(struct snd_pcm_substream *substream,
 
 	adata = dev_get_drvdata(component->dev);
 
-	bits_per_sample = params_width(params);
 	sample_rate = params_rate(params);
 	active_ch = params_channels(params);
 	if (active_ch > stream_data->ch_limit)
-- 
2.31.1

